Astral Pathfinder/Models/Text.hpp

```diff
@@ -0,0 +1,46 @@
+//
+//  Text.hpp
```

Comment:
Should probably be refashioned as PlanetInfo class, where it manages the textBoxes and UI controls associated with planet gameplay information

Reply:
elaborate on your vision for this just a bit more